form.js

```diff
@@ -1,22 +1,37 @@
-const form = document.createElement('form');
+function createElement(tag, attributes = {}) {
+    const element = document.createElement(tag);
+    for (const key in attributes) {
+        element.setAttribute(key, attributes[key]);
+    }
+    console.log(element);
```

Comment:
Violating single responsibility principle.
Create a separate function for adding attributes.  Don't try to create a Swiss knife.